### DISCUSSION TITLE
防撤回出现问题

### DISCUSSION THREAD

**Discussion by c24011** [huiyadanli/RevokeMsgPatcher#210] [Q&A]
![image](https://user-images.githubusercontent.com/88263384/127760427-d24788ba-d2f7-4181-a3a4-c0d0cc89765e.png)
点击后成了这个样子，求解决

**Comment by huiyadanli** [comment]
[新版本](https://github.com/huiyadanli/RevokeMsgPatcher/releases/tag/1.1) 已经支持
